include/seqan3/utility/container/aligned_allocator.hpp

```diff
@@ -176,10 +181,24 @@ class aligned_allocator
     void deallocate(pointer const p, size_type const n) const noexcept
     {
         size_t bytes_to_deallocate = n * sizeof(value_type);
+
+        // Clang doesn't have __cpp_sized_deallocation defined by default even though this is a C++14! feature
+        // > In Clang 3.7 and later, sized deallocation is only enabled if the user passes the `-fsized-deallocation`
+        // > flag.
+        // see also https://clang.llvm.org/cxx_status.html#n3778
```

Comment:
Isn't this something we could configure?

But it also says the following, so I'm not sure.
> The user must supply definitions of the sized deallocation functions, either by providing them explicitly or by using a C++ standard library that does. libstdc++ added these functions in version 5.0, and libc++ added them in version 3.7.

Reply:
No we can't within the source code itself. We could add this to our cmake files, but we would when dictate our users to use that flag. And any other build system would not know about this. I think the safest bet is to just use it if it is enabled, because the one who enables it, must make sure that the provided stdlib (in clang it might the one from gcc or clang) provides those functions, and it is not our liability.